### ISSUE TITLE
is this still under maintenance ?

### ISSUE THREAD

**Issue by bufferoverflow** [CyberNinjas/pam_aad#64]
I just seen https://github.com/aad-for-linux/pam_aad beside of this one and I'm unsure on which one to choose and if one of those is already on the way to be packaged within the Distribution.

**Comment by oxr463** [comment]
Yes, it's moved to here: https://aad-for-linux.github.io/aad-for-linux

**Comment by bufferoverflow** [comment]
thanks for the update @oxr463 ! Would be cool if you could add a note here to the README.md and maybe archive the repo so people are aware of this fact.

**Comment by oxr463** [comment]
I'm not sure who is still around that has access to update the README.md file.
